docs/tutorial.rst

```diff
@@ -121,22 +125,26 @@ There is a preferred way to resolve this. If any kind of fitting object has been
 containing an instance of ``Model``. This can again be called::
 
     model = {y_1: a * x**2, y_2: 2 * x * b}
-    fit = Fit(model, x=xdata)
+    fit = Fit(model, x=xdata, y_1=y_data1, y_2=y_data2)
     fit_result = fit.execute()
 
     y_1, y_2 = fit.model(x=xdata, **fit_result.params)
 
-This returns a tuple with the components evaluated so through the magic of tuple unpacking``y_1`` and ``y_2`` contain the
-evaluated fit. Nice!
+This returns a named tuple with the components evaluated so through the magic of tuple unpackingi ``y_1`` and ``y_2`` contain the
+evaluated fit. Nice! It is usually beter to do the unpacking explicitely though.
 
 If for some reason no ``Fit`` is initiated you can make a Model object yourself::
 
     from symfit import Model
 
     model_dict = {y_1: a * x**2, y_2: 2 * x * b}
-    model = Model.from_dict(model_dict)
+    model = Model(model_dict)
+
+    outcome = fit.model(x=xdata, a=2.4, b=0.1)
+    y_1 = outcome.y_1
+    y_2 = outcome.y_2
 
-    y_1, y_2 = fit.model(x=xdata, a=2.4, b=0.1)
+.. examples above overwrites the variables y_1 and y_2. Plus, the parameters and variables are not defined anywhere...
```

Comment:
yeah the overwriting could be changed perhaps. This line is actually only meant to show how to make a `Model` from a dict. Could perhaps be turned into a fully working example, but I feel that this far into the docs a user should know how to do that right?